app/webpacker/components/Panel/pages/RunValidatorsPage/RunValidatorsForm.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import {
+  Form, FormField, FormGroup, Header, HeaderSubheader, Radio,
+} from 'semantic-ui-react';
+import { useMutation } from '@tanstack/react-query';
+import useInputState from '../../../../lib/hooks/useInputState';
+import { ALL_VALIDATORS, VALIDATORS_WITH_FIX } from '../../../../lib/wca-data.js.erb';
+import { IdWcaSearch } from '../../../SearchWidget/WcaSearch';
+import SEARCH_MODELS from '../../../SearchWidget/SearchModel';
+import CompetitionRangeSelector from './CompetitionRangeSelector';
+import useCheckboxState from '../../../../lib/hooks/useCheckboxState';
+import runValidatorsForCompetitionList from './api/runValidatorsForCompetitionList';
+import runValidatorsForCompetitionsInRange from './api/runValidatorsForCompetitionsInRange';
+import ValidationOutput from './ValidationOutput';
+import WCAQueryClientProvider from '../../../../lib/providers/WCAQueryClientProvider';
+
+const validatorNameReadable = (validatorName) => validatorName.split('::')[1];
+
+const VALIDATOR_OPTIONS = ALL_VALIDATORS.map((validator) => ({
+  key: validator,
+  text: validatorNameReadable(validator),
+  value: validator,
+}));
+
+const COMPETITION_SELECTION_OPTIONS_TEXT = {
+  manual: 'Pick competition(s) manually',
+  range: 'Competition between dates',
+};
+
+const COMPETITION_SELECTION_OPTIONS = Object.keys(COMPETITION_SELECTION_OPTIONS_TEXT);
+
+export default function Wrapper() {
+  return (
+    <WCAQueryClientProvider>
+      <RunValidatorsForm />
+    </WCAQueryClientProvider>
+  );
+}
+
+function RunValidatorsForm({ competitionIds }) {
+  const [
+    selectedCompetitionSelectionOption,
+    setSelectedCompetitionSelectionOption,
```

Comment:
If the names get so long that you have to break them into two lines, you should probably think about shorter names.